src/DateRangePickerInput/web/DateRangePickerInput.js

```diff
@@ -426,7 +434,6 @@ DateRangePickerInput.defaultProps = {
     from: false,
     to: false,
   },
```

Comment:
you can't ignore defaultProps. That's the reason other stories are breaking since they are getting range as undefined in the constructor that's where defaultProps helps you